beacon-chain/core/helpers/committee.go

```diff
@@ -315,14 +315,6 @@ func ShardDeltaFromCommitteeCount(committeeCount uint64) uint64 {
 //        shard = Shard((shard + SHARD_COUNT - get_shard_delta(state, check_epoch)) % SHARD_COUNT)
 //    return shard
 func StartShard(state *pb.BeaconState, epoch uint64) (uint64, error) {
-	startShard, err := startShardCache.StartShardInEpoch(epoch)
```

Comment:
No longer using the cache ?

Reply:
no see description

Reply:
I was able to get the start shard cache to comply with the new execution function, will add the cache back : )